packages/calcite-components/src/components/popover/popover.tsx

```diff
@@ -300,9 +298,9 @@ export class Popover
     this.setFilteredPlacements();
     connectLocalized(this);
     connectMessages(this);
-    connectOpenCloseComponent(this);
     this.setUpReferenceElement(this.hasLoaded);
     connectFocusTrap(this);
+    onToggleOpenCloseComponent(this);
```

Comment:
This needs to be called only when `open = true`, no?

Reply:
yes!